.azure-pipelines/client.yml

```diff
@@ -30,17 +33,40 @@ jobs:
   - task: PublishBuildArtifacts@1
     displayName: 'Publish'
 
-- template: client.test.yml
-  parameters:
-    name: Linux
-    vmImage: 'ubuntu-16.04'
+- job: 'Test'
+  
+  dependsOn:
+  - 'Build'
 
-- template: client.test.yml
-  parameters:
-    name: macOS
-    vmImage: 'macOS-10.13'
+  strategy:
+    matrix:
+      Linux:
+        os.name: 'Linux'
+        os.vmImage: 'ubuntu-16.04'
+      macOS:
+        os.name: 'macOS'
+        os.vmImage: 'macOS-10.13'
+      Windows:
+        os.name: 'Windows'
+        os.vmImage: 'vs2017-win2016'
+         
+  pool:
+    vmImage: $(os.vmImage)
+
+  steps:
+    - script: |
+        mvn jetty:run-forked -Dorg.slf4j.simpleLogger.defaultLogLevel=error -Dorg.slf4j.simpleLogger.log.org.apache.maven.cli.transfer.Slf4jMavenTransferListener=warn --batch-mode -f $pomFile
```

Comment:
You are going to need to change this to `$(pomFile)` otherwise DevOps will not correctly map it.